haystack/components/rankers/meta_field.py

```diff
@@ -227,12 +259,27 @@ def run(
             return {"documents": documents[:top_k]}
 
         if len(docs_missing_meta_field) > 0:
-            logger.warning(
-                "The parameter <meta_field> is currently set to '{meta_field}' but the Documents with IDs {document_ids} don't have this meta key.\n"
-                "These Documents will be placed at the end of the sorting order.",
-                meta_field=self.meta_field,
-                document_ids=",".join([doc.id for doc in docs_missing_meta_field]),
-            )
+            if missing_meta == "bottom":
+                logger.warning(
+                    "The parameter <meta_field> is currently set to '{meta_field}' but the Documents with IDs {document_ids} don't have this meta key.\n"
```

Comment:
Looks like this string can be extracted into a variable and reused in all three warnings?